crates/ironrdp-tokio/src/reqwest.rs

```diff
@@ -37,7 +37,7 @@ impl Default for ReqwestNetworkClient {
 }
 
 impl ReqwestNetworkClient {
-    pub async fn send<'a>(&'a mut self, request: &'a sspi::generator::NetworkRequest) -> ConnectorResult<Vec<u8>> {
+    async fn send_impl<'a>(&'a mut self, request: &'a sspi::generator::NetworkRequest) -> ConnectorResult<Vec<u8>> {
```

Comment:
issue: This is a breaking change, and we need the non-trait-method one in Devolutions Gateway: https://github.com/Devolutions/IronRDP/pull/859

Not sure of what would be a better name here though… Maybe `send_request` or `execute`?

Reply:
`send_request` looks good to me

Reply:
Done